### ISSUE TITLE
The file stop fetching for some video which stop the entrier 

### ISSUE THREAD

**Issue by PkPawarat** [cristminix/LLFetcher#6]
![image](https://github.com/cristminix/LLFetcher/assets/74439556/e004f2c2-6915-4a50-9701-b2948c71e86a)

**Comment by cristminix** [comment]
maybe it cause of unstable network speed

**Comment by sentientlantern** [comment]
I have the same issue and refreshing doesn't resolve it 

![Image](https://github.com/user-attachments/assets/dc1106c8-cd6d-4963-a4be-f4e513186418)

**Comment by Rocco23-dev** [comment]
> Ho lo stesso problema e l'aggiornamento non lo risolve
> 
> ![Image](https://github.com/user-attachments/assets/dc1106c8-cd6d-4963-a4be-f4e513186418)

I also have the same problem

**Comment by N0madS0uL** [comment]
Good day, I am experiencing the same issue on my side, the URL in question is : https://www.linkedin.com/learning/python-for-non-programmers/

![Image](https://github.com/user-attachments/assets/d61c65b6-d9fe-4d25-8f54-8b33daec547f)

I disabled QUIC in Chrome, cleared all cache and cookies and tried again, it simply does not work. For additional context, I usually only use Firefox, so installed Chrome fresh just for this extension, so I cleared it completely and does not make any difference. 

Maybe this could have something to do with the type of video (learning material)? In my case there is a section for CoderPad which maybe is throwing off LLF? I did pull a previous learning video which was a very simple one with no extra features save for a quiz and exam and that worked fine. 

Hope the info helps, good luck and thanks!

**Comment by cristminix** [comment]
I dunno what it goes, i'm dowloading using nypl library account and got this :  [LLFetcher download results python from zeros on 202505011149
](https://youtu.be/AlSpTXwVNow)

thank you for posting this issue

**Comment by N0madS0uL** [comment]
Thanks for the reply, you are at least getting the data pulled, mine stops as per my screenshot. I am using a premium account so I should have complete access. Also why does it only download partially and always stops at the same place. I have only attempted downloading 2 courses so far and the first worked fine. This error show on the extension itself, however I think you are aware of it no?

![Image](https://github.com/user-attachments/assets/9cf42f19-02f4-4ac0-aa67-e3533a3d5b4c)

How do I remove the video completely from LLFetcher? So that I can retry to add it? It seems to keep history of my videos, how do I remove that? I have cleared my browser and cache but it makes no difference. Currently I can't even download or redownload the video.

**Comment by cristminix** [comment]
To reset data you can use this menu:

![Image](https://github.com/user-attachments/assets/5b503d7f-3b85-4671-96ae-ffb71e53a1f0)

**Comment by N0madS0uL** [comment]
Yeah this is not working, I have tried that which you suggest. Furthermore this issue, I think it extends beyond the account type even because I can't even re-download some of the files now. I also can't really afford a premium sub, I only did it because they gave me a free month so I wanted to try it.

![Image](https://github.com/user-attachments/assets/5a749bad-1181-4a4a-a4e8-3fd0144bc0c8)

As per this screenshot, the highlighted ones, basically the first 8 lines I managed to download, but since my first attempt, I can't even download those 8 anymore, only a few of them, but NOTHING beyond the first 8. I also want to completely remove LLF and any history, settings, cache etc so that I can reinstall and retry. Is that possible? Right now I don't see any way of doing it. Using your method does not change anything for me, it shows what I have tried to download before, I want to remove this history and start from a clean slate, is that possible? 

As for the primary issue itself, I can only hope you can address this and solve it. Based on the above comments, I am clearly not the only one that experienced this. Perhaps you should also looking into taking advantage of the free month premium option?

Thanks for the efforts!

**Comment by N0madS0uL** [comment]
Thanks for the link etc to the NYPL card, I tried it, went with option that is outside of US, since I am. I logged in and went straight to the course mentioned, tried it and got further, this time 13 items but then the same issue occurred, which suggests that the account is not the issue. So either it times out at client or server end? Not sure, I am guessing here.

![Image](https://github.com/user-attachments/assets/2eb5f497-e6f4-4f67-9229-d4948eb0b65a)

**Comment by N0madS0uL** [comment]
Just a heads-up, I managed to get this to work, not sure how really, but I will just give you the steps I took.

1. I removed the extension completely
2. Added/installed it again
3. Navigated to the course and added it to the LLE extension for download. 
4. Configured the Queue setup, selected 720p, picked US as language, numbered option was enabled
5. The list was created and I started from the bottom, last item and manually downloaded each.

This seemed to work. I am not sure if this is because too many "connections" were made simultaneously? Maybe its an idea to download each one by one until they complete as it may cause something to fall over? Just a thought, hope the info helps.
